geode-junit/src/main/java/org/apache/geode/test/version/VersionManager.java

```diff
@@ -41,11 +39,9 @@
  * see Host.getVM(String, int)
  */
 public class VersionManager {
-  public static final String CURRENT_VERSION = "000";
-  public static final String GEODE_110 = "110";
-  public static final String GEODE_120 = "120";
-  public static final String GEODE_130 = "130";
-  public static final String GEODE_140 = "140";
+  public static final String CURRENT_VERSION = "0.0.0";
+  public static final String GEODE_130 = "1.3.0";
+  public static final String GEODE_140 = "1.4.0";
```

Comment:
I feel like I asked this before so I apologize if its a repeat...why could we get rid of versions 1.1 and 1.2 here?

Reply:
there were no references to them.  We should probably get rid of GEODE_130 and GEODE_140 as well since the common practice seems to be to hard-code version strings in our tests.